pkg/ingester/wal.go

```diff
@@ -794,7 +795,8 @@ func processWALWithRepair(startSegment int, userStates *userStates, params walRe
 	if err != nil {
 		level.Error(util.Logger).Log("msg", "error in repairing WAL", "err", err)
 	}
-	multiErr := util.NewMultiError(err)
+	multiErr := tsdb_errors.NewMulti()
+	multiErr.Add(err)
 	multiErr.Add(w.Close())
 
 	return multiErr.Err()
```

Comment:
All this can be just `tsdb_errors.NewMulti(err, w.Close()).Err()`, but maybe that can be done in follow up PRs and not required here.

Reply:
Yes, done. Generally speaking I'm not a big fan of such contracted versions (to me it's less readable) but this is your WAL, so your pick ;)